src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/services/service-form/service-form.component.ts

```diff
@@ -215,12 +219,19 @@ export class ServiceFormComponent extends CdForm implements OnInit {
   }
 
   ngOnInit(): void {
+    if (this.router.url.includes('services')) {
+      this.pageURL = 'services';
+    }
     this.action = this.actionLabels.CREATE;
     this.cephServiceService.getKnownTypes().subscribe((resp: Array<string>) => {
       // Remove service types:
       // osd       - This is deployed a different way.
       // container - This should only be used in the CLI.
       this.serviceTypes = _.difference(resp, ['container', 'osd']).sort();
+
+      if (this.wizardStepService.clusterCreation) {
+        this.serviceTypes = _.difference(resp, ['mgr', 'mon', 'crash']).sort();
+      }
```

Comment:
Same thing here: if we find that the Service form needs to hide some specific services depending on who's calling it, let's add that as a input param.